src/frontend/utils/amplitude.ts

```diff
@@ -1,19 +1,17 @@
-import amplitude from 'amplitude-js';
+import * as amplitude from '@amplitude/analytics-browser';
 import { IDokumentasjonsbehov } from '../typer/ettersending';
 
-const amplitudeInstance = amplitude.getInstance();
-
-amplitudeInstance.init('default', '', {
-  apiEndpoint: 'amplitude.nav.no/collect-auto',
-  saveEvents: false,
-  includeUtm: true,
-  includeReferrer: true,
-  platform: window.location.toString(),
+amplitude.init('default', undefined, {
+  serverUrl: 'https://amplitude.nav.no/collect-auto',
+  defaultTracking: false,
```

Comment:
Får vi med referrer? 

sourceName virket litt hacky? Er vel ikke gitt at den slår til heller? 😬

Reply:
Byttet til enklere versjon uten split på ?

Reply:
Det lange svaret: 

In amplitude-js, web attribution is enabled by enabling the following configurations:

config.includeGclid
config.includeFbclid
config.includeReferrer
config.includeUtm
In @amplitude/analytics-browser, the web attribution is controlled by a single configuration config.attribution.disabled which by default is set to false and captures all campaign parameters. This configuration collects the same campaign parameters supported in amplitude-js.


https://www.docs.developers.amplitude.com/data/sdks/typescript-browser/migration/#patterns